src/algoliasearch.helper.js

```diff
@@ -343,6 +343,61 @@ AlgoliaSearchHelper.prototype.getCurrentPage = function() {
   return this.state.page;
 };
 
+/**
+ * Get the list of refinements for a give facet.
```

Comment:
`s/facet/attribute/` I would say, because of the numericFilters and the `_tags` very soon.